### PR TITLE
feature(ui-ux): implement dark mode for OCG

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link:

<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: src/pages/on-chain-governance/index.page.tsx [hunk 1]
```diff
@@ -253,9 +273,9 @@ function UserQueryButtonRow({
           <a
             data-testid="OnChainGovernance.AllProposalsButton"
             className={classNames(
-              "md:font-normal font-medium rounded-l border border-r-0 py-[6px] md:px-[25px] px-3 md:text-base text-xs text-gray-900 border-gray-200",
+              "md:font-normal font-medium rounded-l border border-r-0 py-[6px] md:px-[25px] px-3 md:text-base text-xs text-gray-900 dark:text-dark-gray-900 border-gray-200 dark:border-dark-gray-200",
```
Comment (veralygit) [inline src/pages/on-chain-governance/index.page.tsx:276]: Check bg for buttons for darkmode 
image

image

File: src/pages/on-chain-governance/_components/ProposalCard.tsx [hunk 2]
```diff
@@ -105,16 +100,19 @@ function ProposalCard({
                     data-testid="OnChainGovernance.CardView.ViewButton"
                     onClick={() => setIsViewClicked(!isViewClicked)}
                     className={classNames(
-                      "border border-primary-300 rounded text-primary-400 dark:bg-gray-900 dark:border-gray-700 dark:text-dark-primary-500 px-1.5 py-1 text-sm h-min",
-                      { "bg-primary-100": isViewClicked }
+                      "border-[0.5px] border-primary-300 rounded text-primary-500 dark:text-dark-primary-500 dark:bg-gray-900 border-primary-300 dark:border-dark-primary-300 px-1.5 py-1 text-sm h-min",
```
Comment (veralygit) [inline src/pages/on-chain-governance/_components/ProposalCard.tsx:103]:
```suggestion
                      "border-[0.5px] border-primary-300 rounded text-primary-500 dark:text-dark-primary-500 dark:border-dark-primary-300 px-1.5 py-1 text-sm h-min",
```

File: src/pages/on-chain-governance/_components/ProposalCard.tsx [hunk 3]
```diff
@@ -105,16 +100,19 @@ function ProposalCard({
...
+                      }
                     )}
                   >
                     VIEW
                   </div>
                 </a>
               </Link>
               <div
-                className="text-primary-500 cursor-pointer dark:bg-gray-900 dark:border-gray-700 dark:text-dark-primary-500 border border-primary-300 rounded h-min"
+                className="text-primary-500 cursor-pointer dark:bg-gray-900 dark:text-dark-primary-500 border-[0.5px] border-primary-300 dark:border-dark-primary-300 rounded h-min"
```
Comment (veralygit) [inline src/pages/on-chain-governance/_components/ProposalCard.tsx:115]:
```suggestion
                className="text-primary-500 cursor-pointer dark:text-dark-primary-500 border-[0.5px] border-primary-300 dark:border-dark-primary-300 rounded h-min"
```

File: src/pages/on-chain-governance/_components/VotesTable.tsx [hunk 4]
```diff
@@ -12,33 +12,22 @@ export function VotesTable({ votes }: { votes }): JSX.Element {
...
-    return "md:bg-red-50 text-red-600";
+    return "md:bg-red-50 md:dark:bg-[#FF483D]/[0.15] text-red-600 dark:text-dark-red-600";
   }
   if (vote === "YES") {
-    return "md:bg-green-50 text-green-600";
+    return "md:bg-green-50 md:dark:bg-[#21E529]/[0.15] text-green-600 dark:text-dark-green-600";
   }
   if (vote === "NEUTRAL") {
-    return "md:bg-gray-50 text-gray-600";
+    return "md:bg-gray-50 md:dark:bg-dark-gray-200 text-gray-600 dark:text-dark-gray-600";
```
Comment (kyleleow) [inline src/pages/on-chain-governance/_components/VotesTable.tsx:30]: check text color again

**Review by kyleleow — APPROVED** [review]

**Pull request merged**